File: lottoMaster/app/src/main/java/lottomaster/lunastratos/com/lottomaster/fragment/HomeFragment.java
package lottomaster.lunastratos.com.lottomaster.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.util.Log;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import java.util.HashMap;

import lottomaster.lunastratos.com.lottomaster.LottoInfoAsyncTask;
import lottomaster.lunastratos.com.lottomaster.R;

public class HomeFragment extends Fragment {

    TextView insert_home;
    HashMap map;

    public HomeFragment(HashMap map) {
        this.map = map;
    }


    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
        View v = inflater.inflate(R.layout.fragment_home,container,false);
        insert_home = v.findViewById(R.id.insert_home);

        Log.i("HomeFragment", map.toString());
        if(map !=null){
            insert_home.setText(map.toString());
        }else{
            getNowLottoInfo();
        }


        return v;

    }

    public void getNowLottoInfo(){
        String [] nowLottoAddress = {"https://www.dhlottery.co.kr/gameResult.do?method=byWin", "now"};
        LottoInfoAsyncTask task = new LottoInfoAsyncTask(insert_home);
        task.execute(nowLottoAddress);
    }


}

File: WhiteAlbum/app/src/main/java/memokotlin/lunastratos/com/whitealbum/Fragment/HomeFragment.kt
package memokotlin.lunastratos.com.whitealbum.Fragment


import android.os.Bundle
import android.support.design.widget.TabLayout
import android.support.v4.app.Fragment
import android.support.v4.view.ViewPager
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import memokotlin.lunastratos.com.whitealbum.Adapter.HomeViewAdapter
import memokotlin.lunastratos.com.whitealbum.R

class HomeFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        val v: View = inflater.inflate(R.layout.fragment_home, container, false)

        var mHomeTabLayout = v.findViewById<TabLayout>(R.id.mHomeTabLayout)
        mHomeTabLayout.addTab(mHomeTabLayout.newTab().setText("메인"))
        mHomeTabLayout.addTab(mHomeTabLayout.newTab().setText("내 취향"))

        var mHomeViewPager = v.findViewById<ViewPager>(R.id.mHomeViewPager)
        mHomeViewPager.adapter = HomeViewAdapter(activity!!.supportFragmentManager,mHomeTabLayout.tabCount)
        mHomeViewPager.addOnPageChangeListener(TabLayout.TabLayoutOnPageChangeListener(mHomeTabLayout))

        mHomeTabLayout.addOnTabSelectedListener(object: TabLayout.OnTabSelectedListener{
            override fun onTabReselected(tab: TabLayout.Tab?) {

            }

            override fun onTabUnselected(tab: TabLayout.Tab?) {

            }

            override fun onTabSelected(tab: TabLayout.Tab?) {
                mHomeViewPager.currentItem = tab!!.position
            }
        })
        return v
    }
}